packages/plugin-odoo/src/NoteEditableXmlDomParser.ts

```diff
@@ -0,0 +1,44 @@
+import { XmlDomParsingEngine } from '../../plugin-xml/src/XmlDomParsingEngine';
+import { DividerNode } from '../../plugin-divider/src/DividerNode';
+import { nodeName } from '../../utils/src/utils';
+import { DividerXmlDomParser } from '../../plugin-divider/src/DividerXmlDomParser';
+import { AtomicNode } from '../../core/src/VNodes/AtomicNode';
+
+export class NoteEditableXmlDomParser extends DividerXmlDomParser {
+    static id = XmlDomParsingEngine.id;
+    engine: XmlDomParsingEngine;
+
+    predicate = (item: Node): boolean => {
+        return (
+            item instanceof Element &&
+            nodeName(item) === 'DIV' &&
+            item.classList.contains('note-editable')
+        );
+    };
+
+    async parse(item: Element): Promise<DividerNode[]> {
+        const divider = (await super.parse(item))[0];
+
+        let looseChilds: VNode[] = [];
+        const newChilds: VNode[] = [];
+        for (const child of divider.childVNodes) {
+            if (child instanceof AtomicNode) {
+                looseChilds.push(child);
+            } else {
+                this.addChildren(newChilds, looseChilds);
+                looseChilds = [];
+                newChilds.push(child);
+            }
+        }
+        this.addChildren(newChilds, looseChilds);
+        divider.append(...newChilds);
+        return [divider];
+    }
```

Comment:
I don't like this construction but the alternatives I came up with were not necessarily better so we'll leave it at that and carry on. However, "childs" does not exist in english. The plural form of "child" is "children".

Reply:
done